applications/nrf5340_audio/src/audio/streamctrl.c

```diff
@@ -124,12 +124,12 @@ static void le_audio_rx_data_handler(uint8_t const *const p_data, size_t data_si
 	struct ble_iso_data *iso_received = NULL;
 
 #if (CONFIG_AUDIO_DEV == GATEWAY)
+	static uint32_t packet_count_r;
 	switch (channel_index) {
```

Comment:
Is it necessary to move this?
If so, it should be a newline before the switch

Reply:
Certain versions of the compiler complains, and I believe rightly so, if a variable is declared within a switch case. We either need to move the declaration or add { in the case